### PR TITLE
mfreitassm/c02-network06

### PR DESCRIPTION
**Pull request by mfreitassm**
# Description

Exercise (e.g. C01-E01): 
c02-network06

Student Name:
Mariana Freitas

> Answer to this exercise.

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/02class/exercises/c02-network06/mfreitassm/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# c02-network06
+
+## Commands Execution Output
+
+- Commands for creating the NAT gateway:
+```
+Create a network interface
+//subnet public-b
+aws ec2 create-network-interface --subnet-id subnet-0dd087be76dd34768 --description "devopsacademy network interface" --groups sg-021fa8202867b5ce0
```
Comment (roperto) [inline classes/02class/exercises/c02-network06/mfreitassm/README.md:9]: Hi, no need to create a network interface for this exercise.

File: classes/02class/exercises/c02-network06/mfreitassm/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        "SourceDestCheck": true,
+        "Status": "pending",
+        "SubnetId": "subnet-0dd087be76dd34768",
+        "TagSet": [],
+        "VpcId": "vpc-04426efcf5b2228a4"
+    }
+}
+
+Associate Network Interface with te elastic IP
+aws ec2 associate-address --allocation-id eipalloc-0ce4bab5ca01b643f --network-interface-id eni-06bc2518d39cdf022 --private-ip-address 10.0.10.174
```
Comment (roperto) [inline classes/02class/exercises/c02-network06/mfreitassm/README.md:44]: where does `eipalloc-0ce4bab5ca01b643f` comes from?

File: classes/02class/exercises/c02-network06/mfreitassm/README.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+{
+    "AssociationId": "eipassoc-0b36d9fb021c930ed"
+}
+
+Create NAT
+aws ec2 create-nat-gateway --subnet-id subnet-0dd087be76dd34768 --allocation-id eipalloc-0ce4bab5ca01b643f --tags Key=Name,Value=devopsacademy-ngw
```
Comment (roperto) [inline classes/02class/exercises/c02-network06/mfreitassm/README.md:50]: The way to tag during creating is `--tag-specifications 'ResourceType=natgateway,Tags=[{Key=Name,Value=devopsacademy-ngw}]'`

But you can also tag it like you did below, no problems.

**Comment by roperto** [comment]
It looks good, you probably did the missing command but forgot to paste in the answers.

File: classes/02class/exercises/c02-network06/mfreitassm/README.md (hunk 1 above)
Comment (mfreitassm) [reply classes/02class/exercises/c02-network06/mfreitassm/README.md]: Okay

File: classes/02class/exercises/c02-network06/mfreitassm/README.md (hunk 2 above)
Comment (mfreitassm) [reply classes/02class/exercises/c02-network06/mfreitassm/README.md]: I forgot to put on the file.

File: classes/02class/exercises/c02-network06/mfreitassm/README.md (hunk 3 above)
Comment (mfreitassm) [reply classes/02class/exercises/c02-network06/mfreitassm/README.md]: Copy, Thanks!

**Pull request merged**